spec/menu_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'menu'
+
+describe Menu do
+
+  subject(:menu) { described_class.new(food) }
+
+  let(:food) do
+    {
+      wings: 5.00,
```

Comment:
Props for adding the challenge of using floats. You're the only person I've seen so far using accurate pricing rather than just an integer